gradle.properties

```diff
@@ -1,5 +1,5 @@
 version = 7.3.4-SNAPSHOT
 group = com.graphql-java-kickstart
 
-LIB_GRAPHQL_JAVA_VER = 11.0
+LIB_GRAPHQL_JAVA_VER = 12.0
```

Comment:
Is this really all that's necessary? Everything else is just compatible as-is already?

Reply:
I haven't tested it out yet.  I can see about forcing v12 in my Dropwizard example project at https://github.com/smoketurner/dropwizard-graphql/tree/master/graphql-example to see if any changes are necessary.

Reply:
Sorry for the delay, been tied up with other projects to finish this up.